lib/grpc/GrpcService.ts

```diff
@@ -382,17 +386,45 @@ class GrpcService {
   }
 
   /*
-   * See [[Service.subscribePeerOrders]]
+   * See [[Service.subscribeAddedOrders]]
+   */
+  public subscribeAddedOrders: grpc.handleServerStreamingCall<xudrpc.SubscribeAddedOrdersRequest, xudrpc.Order> = (call) => {
+    this.service.subscribeAddedOrders((order: StampedOrder) => {
+      call.write(createOrder(order));
+    });
+  }
+
+  /*
+   * See [[Service.subscribeRemovedOrders]]
    */
-  public subscribePeerOrders: grpc.handleServerStreamingCall<xudrpc.SubscribePeerOrdersRequest, xudrpc.SubscribePeerOrdersResponse> = (call) => {
-    this.service.subscribePeerOrders((order: StampedPeerOrder) => call.write({ order }));
+  public subscribeRemovedOrders: grpc.handleServerStreamingCall<xudrpc.SubscribeRemovedOrdersRequest, xudrpc.OrderRemoval> = (call) => {
+    this.service.subscribeRemovedOrders((order: OrderPortion) => {
+      const orderRemoval = new xudrpc.OrderRemoval();
+      orderRemoval.setPairId(order.pairId);
+      orderRemoval.setOrderId(order.orderId);
+      orderRemoval.setQuantity(order.quantity);
+      orderRemoval.setLocalId(order.localId || '');
+      orderRemoval.setIsOwnOrder(!!order.localId);
```

Comment:
The `!!order.localId` markup is still confusing me. What about `(order.localId === undefined)` for the sake of readability?